lib/play_games.dart

```diff
@@ -339,4 +352,30 @@ class PlayGames {
     }
     return result;
   }
+
+  static Future<SigninResult> getLastSignedInAccount() async {
+    final Map<dynamic, dynamic> map =
+        await _channel.invokeMethod('getLastSignedInAccount');
+    SigninResultType type = _typeFromStr(map['type']);
+    SigninResult result = new SigninResult()..type = type;
+    if (type != SigninResultType.NOT_SIGNED_IN) {
+      if (type == SigninResultType.SUCCESS) {
+        result.account = new Account()
+          ..id = map['id']
+          ..displayName = map['displayName']
+          ..email = map['email']
+          ..hiResImageUri = map['hiResImageUri']
+          ..iconImageUri = map['iconImageUri'];
+      } else {
+        result.message = map['message'];
+      }
+    }
+    return result;
+  }
+
+  static Future<Void> signOut() async {
+    final Map<dynamic, dynamic> map = await _channel.invokeMethod('signOut');
+
+    return Void();
```

Comment:
is this really necessary? I assume you can just do:
```
  static Future<Void> signOut() async {
    await _channel.invokeMethod('signOut');
}
```
or (also good, not sure if it works)
```
```
  static Future<Void> signOut() {
    return _channel.invokeMethod('signOut');
}
```